site/content/en/docs/installation/_index.md

```diff
@@ -79,27 +79,27 @@ cd lws
 IMAGE_REGISTRY=<registry>/<project> make image-push deploy
 ```
 
-## Uninstall
+### Uninstall
 
 To uninstall LeaderWorkerSet, run the following command:
 
 ```sh
 make undeploy
 ```
 
-# Install in a different namespace
+## Install in a different namespace
 
 To install the leaderWorkerSet controller in a different namespace rather than `lws-system`, you should first:
 ```sh
 git clone https://github.com/kubernetes-sigs/lws.git
 cd lws
 ```
-Then change the [kustomization.yaml](../../config/default/kustomization.yaml) _namespace_ field as:
+Then change the [kustomization.yaml](https://github.com/kubernetes-sigs/lws/blob/main/config/default/kustomization.yaml) _namespace_ field as:
 ```yaml
 namespace: <your-namespace>
 ```
 
-# Optional: Use cert manager instead of internal cert
+## Optional: Use cert manager instead of internal cert
 The webhooks use an internal certificate by default. However, if you wish to use cert-manager (which
 supports cert rotation), instead of internal cert, you can by performing the following steps.
 
```

Comment:
Does the `# Install with helm chart` need change to `## Install with helm chart`